### ISSUE TITLE
[Question] Torrent Sources

### ISSUE THREAD

**Issue by TwilightMercy** [debridmediamanager/debrid-media-manager#122]
Hi, I wanted to ask,

What torrent websites does DMM search to find cached torrents on AD/RD services?

**Comment by yowmamasita** [comment]
Yes this is correct, it scrapes about 15 websites to gather all hashes that is possibly cached in Real-Debrid.

I don't remember the complete list but here are some (it's a combination of private and public trackers) - 1337x, rarbg, torrent galaxy, YTS, Nyaa.si, TorrentLeech, Milkie

**Comment by TwilightMercy** [comment]
Great to hear, thank you. 

I have another question - 

Can I use DMM as a scraper source? (Similar to using Torrentio as a scraper source)

Does DMM have API for that?

**Comment by TwilightMercy** [comment]
Example:
https://debridmediamanager.com/api/torrents/movie?imdbId=tt15398776

but without auth : )

**Comment by yowmamasita** [comment]
Answering the question and closing. DMM searches its own scraped index plus Torrentio, Comet, MediaFusion, Peerflix and TorrentsDB. Each of those is a toggle in Settings, so you can turn off any source you do not want.
